pages/Components/Navbar/NavbarOptions.tsx

```diff
@@ -13,15 +20,30 @@ export const NavbarOptions = ({activeBurger}:any) => {
         </button>
       </form>
 
-      <div className=" flex flex-wrap font-bold lg:ml-20">
-        <img
-          className="rounded-full max-w-[40px]  m-2"
-          src="/images/user.jpg"
-        />
-        <p className="my-auto">
-          <a className=" p-2">Miguel Lopez</a>
-        </p>
-      </div>
+      {session ? (
```

Comment:
Better to use `{session && `
and `{!session && ` instead of ternary operator "?"